kafka/pykafka/connection.py

```diff
@@ -74,10 +75,8 @@ def response(self):
         try:
             size = self._socket.recv(4)
             size = struct.unpack('!i', size)[0]
-            output = bytearray(size)
-            recvall_into(self._socket, output)
-            # TODO: Figure out if correlation ids are worth it
```

Comment:
I didn't look into it actually, I just removed it because there are better places for that reminder.